keyboards/lazydesigners/the30/rules.mk

```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+#
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
+EXTRAKEY_ENABLE = yes        # Audio control and System control
+NKRO_ENABLE = yes            # USB Nkey Rollover
```

Comment:
I think I'd prefer if this file contained all the rules that are set in the [AVR template](https://github.com/qmk/qmk_firmware/blob/b608bddc5edae7ccce8108d9c7777437160f3cb3/quantum/template/avr/rules.mk#L24-L42), even if said rules are set to `no`. Right off the bat I can see from the Lazy Designers website that the PCB supports Backlight and their stock firmware has it enabled.

Reply:
Good point. Fixed.